web/src/lib/components/shared-components/search-bar/search-bar.svelte

```diff
@@ -122,11 +122,11 @@
           ? 'rounded-t-3xl border  border-gray-200 bg-white dark:border-gray-800'
           : 'rounded-3xl border border-transparent bg-gray-200'}"
         placeholder="Search your photos"
-        required
```

Comment:
Removing `required` is an issue, because:
1. Empty searches are included in recent searches
2. The server request fails when searching for an empty string

Reply:
Good to know, thanks for catching this!